Error compiling on esphome 2025.2.0

I got those errors:

INFO ESPHome 2025.2.0
INFO Reading configuration /config/esphome/aa-mitsubishi-dormitorioprincipal.yaml...
INFO Detected timezone 'Europe/Madrid'
INFO Generating C++ source...
INFO Compiling app...
Processing aa-mitsubishi-dormprincipal (board: d1_mini; framework: arduino; platform: platformio/espressif8266@4.2.1)
--------------------------------------------------------------------------------
HARDWARE: ESP8266 80MHz, 80KB RAM, 4MB Flash
Dependency Graph
|-- ESPAsyncTCP-esphome @ 2.0.0
|-- ESPAsyncWebServer-esphome @ 3.2.2
|-- DNSServer @ 1.1.1
|-- ESP8266WiFi @ 1.0
|-- ESP8266mDNS @ 1.2
|-- noise-c @ 0.1.6
Compiling .pioenvs/aa-mitsubishi-dormprincipal/src/esphome/components/cn105/climateControls.cpp.o
Compiling .pioenvs/aa-mitsubishi-dormprincipal/src/esphome/components/cn105/cn105.cpp.o
Compiling .pioenvs/aa-mitsubishi-dormprincipal/src/esphome/components/cn105/componentEntries.cpp.o
Compiling .pioenvs/aa-mitsubishi-dormprincipal/src/esphome/components/cn105/cycle_management.cpp.o
In file included from src/esphome.h:39,
                 from src/esphome/components/cn105/Globals.h:2,
                 from src/esphome/components/cn105/cn105.h:2,
                 from src/esphome/components/cn105/climateControls.cpp:1:
src/esphome/components/cn105/uptime_connection_sensor.h:2:10: fatal error: esphome/components/uptime/uptime_seconds_sensor.h: No such file or directory
    2 | #include "esphome/components/uptime/uptime_seconds_sensor.h"
      |          ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
compilation terminated.
*** [.pioenvs/aa-mitsubishi-dormprincipal/src/esphome/components/cn105/climateControls.cpp.o] Error 1
In file included from src/esphome.h:39,
                 from src/esphome/components/cn105/Globals.h:2,
                 from src/esphome/components/cn105/cn105.h:2,
                 from src/esphome/components/cn105/cn105.cpp:2:
src/esphome/components/cn105/uptime_connection_sensor.h:2:10: fatal error: esphome/components/uptime/uptime_seconds_sensor.h: No such file or directory
    2 | #include "esphome/components/uptime/uptime_seconds_sensor.h"
      |          ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
compilation terminated.
*** [.pioenvs/aa-mitsubishi-dormprincipal/src/esphome/components/cn105/cn105.cpp.o] Error 1
In file included from src/esphome.h:39,
                 from src/esphome/components/cn105/Globals.h:2,
                 from src/esphome/components/cn105/cn105.h:2,
                 from src/esphome/components/cn105/componentEntries.cpp:1:
src/esphome/components/cn105/uptime_connection_sensor.h:2:10: fatal error: esphome/components/uptime/uptime_seconds_sensor.h: No such file or directory
    2 | #include "esphome/components/uptime/uptime_seconds_sensor.h"
      |          ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
compilation terminated.
*** [.pioenvs/aa-mitsubishi-dormprincipal/src/esphome/components/cn105/componentEntries.cpp.o] Error 1
In file included from src/esphome.h:39,
                 from src/esphome/components/cn105/Globals.h:2,
                 from src/esphome/components/cn105/cn105.h:2,
                 from src/esphome/components/cn105/cycle_management.cpp:2:
src/esphome/components/cn105/uptime_connection_sensor.h:2:10: fatal error: esphome/components/uptime/uptime_seconds_sensor.h: No such file or directory
    2 | #include "esphome/components/uptime/uptime_seconds_sensor.h"
      |          ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
compilation terminated.
*** [.pioenvs/aa-mitsubishi-dormprincipal/src/esphome/components/cn105/cycle_management.cpp.o] Error 1
========================== [FAILED] Took 1.09 seconds ==========================


For your info, do not update esphome

--- a/components/cn105/uptime_connection_sensor.h
+++ b/components/cn105/uptime_connection_sensor.h
@@ -1,6 +1,5 @@
 #pragma once
-#include "esphome/components/uptime/uptime_seconds_sensor.h"
-
+#include <esphome/components/uptime/sensor/uptime_seconds_sensor.h>
 namespace esphome {
     namespace uptime {
 

